Lib/tkinter/__init__.py

```diff
@@ -7,25 +7,25 @@
 LabelFrame and PanedWindow.
 
 Properties of the widgets are specified with keyword arguments.
-Keyword arguments have the same name as the corresponding resource
+Keyword arguments have the same name as the corresponding options
 under Tk.
 
 Widgets are positioned with one of the geometry managers Place, Pack
 or Grid. These managers can be called with methods place, pack, grid
 available in every Widget.
 
-Actions are bound to events by resources (e.g. keyword argument
-command) or with the method bind.
+Actions are bound to events by options (e.g. the command
```

Comment:
Either this, or "command" should be a docs cross-reference.

```suggestion
Actions are bound to events by options (e.g., the *command*
```

Reply:
Posted this before seeing the discussion about omitting formatting from docstrings. This should be consistent with whatever is decided.